src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.cs

```diff
@@ -15,39 +15,39 @@ namespace System.Text
     public static partial class PrimitiveFormatter
     {
         #region Integers
-        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format, EncodingData encoding)
+        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format = default(TextFormat), EncodingData encoding = default(EncodingData))
         {
             return IntegerFormatter.TryFormatUInt64(value, 1, buffer, out bytesWritten, format, encoding);
         }
 
-        public static bool TryFormat(this sbyte value, Span<byte> buffer, out int bytesWritten, TextFormat format, EncodingData encoding)
+        public static bool TryFormat(this sbyte value, Span<byte> buffer, out int bytesWritten, TextFormat format = default(TextFormat), EncodingData encoding = default(EncodingData))
         {
```

Comment:
I thought the guideline was to have out parameters be at the end. Is that not the case? On the other hand, I also thought that default parameters go in the end. Is this the recommended approach when you have both?